docusaurus/docs/user-docs/content-manager/saving-and-publishing-content.md

```diff
@@ -42,6 +42,22 @@ Published contents can be unpublished, switching back to being drafts again.
 
 To unpublish content, click on the **Unpublish** button in the top right corner of the content editor.
 
+### Bulk publish and unpublish <BetaBadge />
```

Comment:
The feature is currently only released as an alpha but I don't know if we have a tag for that

Reply:
We do, I just wasn't sure if the documentation would be released for the alpha or the beta haha
(I'll replace with `<AlphaBadge />` when necessary)